src/main/scala/common/consts.scala

```diff
@@ -27,7 +27,7 @@ trait BOOMDebugConstants
   val DEBUG_PRINTF        = false // use the Chisel printf functionality
   val COMMIT_LOG_PRINTF   = false // dump commit state, for comparision against ISA sim
   val MEMTRACE_PRINTF     = false // dump trace of memory accesses to L1D for debugging
-  val O3PIPEVIEW_PRINTF   = false // dump trace for O3PipeView from gem5
+  val O3PIPEVIEW_PRINTF   = true // dump trace for O3PipeView from gem5
```

Comment:
I'm not sure if we should include this in the branch - but doesn't really matter